docs/development/remote-development.md

```diff
@@ -32,27 +37,33 @@ You can use [Gitpod](https://gitpod.io/) for light development work like:
 - Editing the docs
 - Running ESLint, Prettier
 
-For proper development use GitHub Codespaces.
+For proper development, use GitHub Codespaces.
 
 The config file for Gitpod is `.gitpod.yml` in the root of the repository.
 
+Gitpod has comes with 50 free hours each month.
+If you need more hours you'll need to buy a plan with more hours.
+
 ### Gitpod tips
 
-- Use `yarn jest:16` to run the tests on Gitpod.
+- Use `yarn jest:16` to run the tests on Gitpod
 
 ### Known problems with Gitpod
 
-`yarn jest:16` has some failing tests.
+- `yarn jest:16` has some failing tests
+- You can't preview Markdown files
 
 ## GitHub Codespaces
 
 The Renovate developers use [GitHub Codespaces](https://github.com/features/codespaces).
 The config files are in the `.devcontainer` folder in the repository.
 
+You can only use GitHub Codespaces if you're a member of the Renovate organization, and have permission to use Codespaces.
+
 ### Codespaces tips
 
-List tips for codespaces here.
+List the tips for codespaces here.
```

Comment:
Do you have tips for using Codespaces?

Reply:
Let's leave it out for now so we can merge this doc earlier rather than later